Documentation/troubleshooting.md

```diff
@@ -136,3 +136,11 @@ spec:
 ```
 
 The incorrect example will give an error along these lines `spec.endpoints.port in body must be of type string: "integer"`
+
+#### Prometheus/Alertmanager pods stuck in terminating loop with healthy start up logs
+
+Double check that only one operator is running on the cluster! Having more than one operator wil lead to such beahaviour. To check run:
```

Comment:
Nit: Check that only one operator is running on the cluster! Having more than one operator will lead to such behaviour. To check run: